app/helpers/recruitment_cycle_helper.rb

```diff
@@ -48,4 +48,8 @@ def current_cycle_provider(provider)
   def next_cycle_provider(provider)
     RecruitmentCycle.next.providers.find_by(provider_code: provider.provider_code)
   end
+
+  def recruitment_cycle_after_2024_and_tda_active?(provider_or_course)
```

Comment:
Can we rename to just tda_active? or tda_feature? or tda_feature_active?

Reply:
Yes, we can. Done: https://github.com/DFE-Digital/publish-teacher-training/pull/4202/commits/88e4d1b2b13eed099200c83bd1a7c02f6a815ce9